extension-workspace-reference.md

```diff
@@ -0,0 +1,121 @@
+# workspace/xreference extension to LSP
```

Comment:
Interesting argument! Well, I also like `workspace/references` better and only did it to align with `workspace/symbol`.. since your argument sounds pretty logical I'll update it to the better-liked `workspace/xreferences` :)